tests/levensthein-distance.test.js

```diff
@@ -0,0 +1,19 @@
+const test = require('ava');
+const {LevenshteinDistanceWithMax} = require('../lib/levensthein-distance');
+
+test("shouldn't match", t => {
+  t.is(LevenshteinDistanceWithMax('water', 'atect', 2), false);
+  t.is(LevenshteinDistanceWithMax('water', 'christmas', 3), false);
+  t.is(LevenshteinDistanceWithMax('water', 'water1', 0), false);
+  t.is(LevenshteinDistanceWithMax('thea', 'ythee', 1), false);
+  t.is(LevenshteinDistanceWithMax('12345', '567', 4), false);
+  t.is(LevenshteinDistanceWithMax('var', 'abc', 3), false);
```

Comment:
This should be true. You only have to do 3 substitutions from `var` to `abc`.

Reply:
That's correct, but knowinlgy if you make an typo like `abc` I would be surprised you meant `var`. But we can make it `true`